content-service/src/main/java/io/meeds/news/rest/NewsRest.java

```diff
@@ -354,14 +354,17 @@ public ResponseEntity<News> getNewsById(@PathVariable("id")
                                           boolean editMode,
                                           @Parameter(description = "article translation")
                                           @RequestParam(name = "lang", required = false)
-                                          String lang) {
+                                          String lang,
+                                          @Parameter(description = "Fetch the default article if a translation does not exist for the given language")
+                                          @RequestParam(name = "getDefault", defaultValue = "false", required = false)
```

Comment:
to be removed IMO, to keep only for service layer if the value can be different
-> fetchOriginal

Reply:
done !